yt/yt/tests/integration/proxies/test_rpc_proxy.py

```diff
@@ -853,6 +853,39 @@ def test_sort(self):
 
         assert select_rows("* from [//tmp/t_out] LIMIT " + str(2 * size)) == original_table
 
+    @authors("faucct")
+    def test_writing_large_rows(self):
```

Comment:
Noticed that this test is in `TestOperationsRpcProxy`, I don't think that is appropriate, operations = MR. Let's either move it somewhere else, or make a separate suite inherited from TestRpcProxyBase. Like TestWireProtocolValidation.

P.S. You are writing large values here (though rows are also large), maybe test_writing_large_values is a better name.